blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: 'Announcing Sourcegraph 3.1'
+author: 'Ryan Blunden'
+publishDate: 2019-02-20T05:59-06:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-3.1
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: false
+---
+
+Sourcegraph 3.1 is now out! This release includes stability and documentation improvements, plus an easy query builder for searching your code (with useful query examples). Also, mark your calendars: our new fixed release schedule means you get a new Sourcegraph release on the 20th of each month.
+
+<!-- (ryan): Hyperlink to headings in above paragraph -->
+
+What is Sourcegraph? Sourcegraph is an open-source, self-hosted, cross-repository code search and navigation tool, with an efficient web interface and "feels-like-native" integration into your code host. [Install or upgrade Sourcegraph.](#install-or-upgrade)
+
+## Find answers in code more easily with the new query builder
+
+<!-- TODO (ryan): Below paragraph needs work as it sounds awkward -->
+
+Did you know that in addition to code, you can search commit diffs, commit messages and symbols?
```

Comment:
```suggestion
Did you know that in addition to code, you can search commit diffs, commit messages, and symbols?
```

Reply:
@ryan-blunden 	did you intentionally discard my suggestions to this section?